sql/api/src/main/scala/org/apache/spark/sql/catalyst/analysis/alreadyExistException.scala

```diff
@@ -31,7 +31,7 @@ class DatabaseAlreadyExistsException(db: String)
   extends NamespaceAlreadyExistsException(Array(db))
 
 // any changes to this class should be backward compatible as it may be used by external connectors
-class NamespaceAlreadyExistsException private(
+class NamespaceAlreadyExistsException private[sql](
```

Comment:
`catalyst` is already an internal package (https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/package.scala#L21-L22) so we can remove `private[sql]`

Reply:
These exception are moved from `sql/catalyst` module to `sql/api` moduel and I am not sure if the assumption still holds for them. Also, according to the comment above:
```
any changes to this class should be backward compatible as it may be used by external connectors
```

It may be safe to keep the `private[sql]`?